docs/docs/personalization.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Personalization
+---
+
+## What is Personalization?
+
+- Have you ever landed on a website that greeted you by name?
+
+- Did you ever search for a book and realized your suggested results based on your previous purchases?
+
+These are examples of personalization. Personalization adjusts the content of a website, and creates a custom experience for users. With personalization, one website can offer a variety of experiences to different visitors.
+
+## Why is Personalization Important?
+
+If you are [creating an e-commerce site](https://www.gatsbyjs.org/tutorial/ecommerce-tutorial/), personalization is key in converting your visitors into buyers. Online merchants use personalization as a way to suggest relevant items to a shopper.
+
+Outside of e-commerce, personalization is important for other types of websites, like e-learning. Personalization, if implemented strategically, can help educational sites deliver more targeted learning experiences. For example, if a learner is struggling with the learning the basics of JavaScript, the website can give the learner additional JavaScript practice problems.
+
+## Using Personalization with Gatsby
+
+Personalization contains dynamic content. Personalization is possible in Gatsby because Gatsby can enhance static content by React Hydration. During build time Gatsby invokes the React server-side DOMS to generate content. You can [learn more about it here](https://www.gatsbyjs.org/blog/2018-10-15-beyond-static-intro/#hydration).
```

Comment:
not shure - small s on plural? or remove the s here?

```suggestion
Personalization contains dynamic content. Personalization is possible in Gatsby because Gatsby can enhance static content by React Hydration. During build time Gatsby invokes the React server-side DOMs to generate content. You can [learn more about it here](https://www.gatsbyjs.org/blog/2018-10-15-beyond-static-intro/#hydration).
```

Reply:
maybe better link text

```suggestion
Personalization contains dynamic content. Personalization is possible in Gatsby because Gatsby can enhance static content by React Hydration. During build time Gatsby invokes the React server-side DOMS to generate content. In this blog post you can learn more about [React Hydration](https://www.gatsbyjs.org/blog/2018-10-15-beyond-static-intro/#hydration).
```

Reply:
Good eye. DOMs is incorrect. The sentence is only referring to one DOM, so the 's' should be deleted.